INIT.md

```diff
@@ -0,0 +1,11 @@
+## Plan
+1. Add option `current_environment` to __Anyway::Settings__ class. Provide default value for this, maybe `ENV["RAILS_ENV"]`?
```

Comment:
It's better to use `Rails.env` for Rails apps. For non-Rails apps we shouldn't set `current_environment` since it could break backward compatibility.